app/views/play/modal/PollModal.coffee

```diff
@@ -77,7 +77,26 @@ module.exports = class PollModal extends ModalView
     pollVotes[@poll.id] = $selectedAnswer.data('answer').toString()
     @userPollsRecord.set 'polls', pollVotes
     @updateAnswers true
-    @userPollsRecord.save {polls: pollVotes}, {success: => @awardRandomGems?()}
+    @userPollsRecord.save {polls: pollVotes}, {success: => 
+      @awardRandomGems?()
+
+      myAnswer = (@userPollsRecord.get('polls') ? {})[@poll.id]
+      answerObj = _.find(@poll.get('answers'), (answer) => answer.key == myAnswer) or {}
+      nextPollId = answerObj.nextPoll
+      
+      # The following block allows for the user to be indecisive with their answer, updating UI accordingly.
+      btn = @$el.find('.btn.btn-illustrated.btn-lg.done-button')
+      if nextPollId
+        btn = @$el.find('.btn.btn-illustrated.btn-lg.done-button')
+        btn.text(i18n.t('common.next'))
+        btn.one('click', ()=>
```

Comment:
`btn.on`?

Reply:
[`one`](https://api.jquery.com/one/) only allows the button to be clicked once, preventing loading the next poll many times and strange behavior.